core-bundle/src/Resources/config/services.yml

```diff
@@ -268,6 +268,13 @@ services:
         tags:
             - { name: data_collector, template: '@ContaoCore/Collector/contao.html.twig', id: contao }
 
+    Contao\CoreBundle\Doctrine\Backup\BackupManager:
+        arguments:
+            - '@database_connection'
+            - '' # Set by extension
+            - [] # Set by extension
+            - 0 # Set by extension
```

Comment:
I would apply our sane default config here as well just to be explicit

Reply:
Done in 72abbf77b357a98f9b483f9b1edda4113ab9bf0f.